detection-rules/link_contains_punycode_characters.yml

```diff
@@ -1,29 +1,30 @@
-name: "Link to a Domain with Punycode Characters"
-description: |
-  The body contains a link to a domain with Punycode characters to hide the true URL destination, or contains non-printable ASCII content.
-references:
-  - "https://www.bleepingcomputer.com/news/security/hackers-abuse-lookalike-domains-and-favicons-for-credit-card-theft/"
-type: "rule"
-authors:
-  - twitter: "ajpc500"
-severity: "medium"
-source: |
-  type.inbound
-  and any(body.links, .href_url.domain.punycode is not null) 
-  and (
-    // include automated emails
-    sender.display_name == "WordPress"
-    or sender.email.local_part == "wordpress"
-
-    // Has my organization ever sent an email to this sender?
-    or (
-        sender.email.domain.root_domain in $free_email_providers
-        and sender.email.email not in $recipient_emails
-    )
-    or (
-        sender.email.domain.root_domain not in $free_email_providers
-        and sender.email.domain.domain not in $recipient_domains
-    )
-  )
-tags:
-  - "Suspicious content"
+name: "Link to a Domain with Punycode Characters"
```

Comment:
Just calling out for future viewers these are rewritten because they have `CRLF` line endings instead of `LF`. Maybe we'll clean this up in the future w/ automation in a PR, but not a big deal for now.